support/mock/mockrt.go

```diff
@@ -212,8 +212,8 @@ func (rt *Runtime) Send(toAddr addr.Address, methodNum abi.MethodNum, params run
 	}
 	expectedMsg := rt.expectSends[0]
 
-	if expectedMsg.Equal(toAddr, methodNum, params, value) {
-		rt.t.Errorf("expectedMessage being sent does not match expectation. Message: to %v method: %v value: %v params: %v, Expected: %v", toAddr, methodNum, value, params, rt.expectSends[0].String())
+	if !expectedMsg.Equal(toAddr, methodNum, params, value) {
+		rt.t.Errorf("expectedMessage being sent does not match expectation. Message: to %v method: %v value: %v params: %v, Expected: %v", toAddr.String(), methodNum, value, params, rt.expectSends[0].String())
```

Comment:
If you explicitly want the `String()` instead of `Format()` output, use `%s`. But I think we should prefer Format() when it's defined.